src/main/java/mate/academy/Main.java

```diff
@@ -32,7 +32,7 @@ public static void main(String[] args) {
 
         MovieSessionService movieSessionService = (MovieSessionService)
                 injector.getInstance(MovieSessionService.class);
-        LocalDate localDate = LocalDate.of(2023, 6, 23);
+        LocalDateTime localDate = LocalDateTime.of(2023, 6, 23, 1, 1);
```

Comment:
It is better to set more realistic time to test data:
`LocalDateTime.of(2023, 6, 23, 16, 30);`